apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -217,6 +220,106 @@ export class ReminderEmailsService {
     return transformedReducedMatches
   }
 
+  async getMentorsAndMenteesWithoutMentoringSessionsLogged({
```

Comment:
This looks quite familiar to another PR where I offered some pseudocode to simplify. What happened to that? Is it reusable here?